src/pages/Ranking.tsx

```diff
@@ -0,0 +1,231 @@
+/* eslint-disable no-console */
+import styled from '@emotion/styled'
+import { useEffect, useState } from 'react'
+import { Duration } from 'luxon'
+import { useNavigate, useParams } from 'react-router'
+import DateSelect from '../components/DateSelect'
+import RankingMock from '../mocks/RankingMock'
+import axiosInstance from '../api/axiosInstance'
+import chatbubble from '../assets/chatbubble.svg'
+
+const Ranking = () => {
+  const { groupId } = useParams()
+  const navigate = useNavigate()
+
+  const [selectedDate, setSelectedDate] = useState(new Date())
+  const [rankData, setRankData] = useState(RankingMock)
+
+  const formatDuration = (isoDuration: string) => {
+    const duration = Duration.fromISO(isoDuration)
+    const hours = String(duration.hours).padStart(2, '0')
+    const minutes = String(duration.minutes).padStart(2, '0')
+    const seconds = String(duration.seconds).padStart(2, '0')
+    return `${hours}:${minutes}:${seconds}`
```

Comment:
```suggestion
    return duration.toFormat('hh:mm:ss')
```

사용하는 라이브러리에서 지원하는 기능이 있는지 먼저 확인한 후에 추가적인 코드를 작성하는게 좋겠습니다. 활용하는 기술의 공식문서를 전체적으로 찬찬히 읽으시고 활용하신다면 조금 더 빠르게 개발하실 수 있을 거 같아요.